test/modules/ModuleLib/TestResults/PriorityQueueTestStatus.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults
+{
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class PriorityQueueTestStatus
+    {
+        public PriorityQueueTestStatus(bool isFinished, int resultCount)
+        {
+            this.IsFinished = Preconditions.CheckNotNull(isFinished, nameof(isFinished));
+            this.ResultCount = Preconditions.CheckNotNull(resultCount, nameof(resultCount));
```

Comment:
`bool` and `int` are both primitives...they're value objects and can never be null.